src/include/catalog/pg_proc.dat

```diff
@@ -5351,6 +5351,15 @@
   proargmodes => '{i,o,o,o,o,o,o,o,o,o,o,o,o,o,o,o,o,o,o,o,o,o,o,o,o,o,o,o,o,o,o}',
   proargnames => '{pid,datid,pid,usesysid,application_name,state,query,wait_event_type,wait_event,xact_start,query_start,backend_start,state_change,client_addr,client_hostname,client_port,backend_xid,backend_xmin,backend_type,ssl,sslversion,sslcipher,sslbits,ssl_client_dn,ssl_client_serial,ssl_issuer_dn,gss_auth,gss_princ,gss_enc,leader_pid,query_id}',
   prosrc => 'pg_stat_get_activity' },
+{ oid => '2173',
+  descr => 'statistics: cumulative information about currently active backends',
+  proname => 'pg_stat_get_session', prorows => '100', proisstrict => 'f',
+  proretset => 't', provolatile => 's', proparallel => 'r',
+  prorettype => 'record', proargtypes => 'int4',
+  proallargtypes => '{int4,float8,int8,float8,int8}',
+  proargmodes => '{i,o,o,o,o}',
```

Comment:
is the first arg (pid) is meant to be only "in", or only "out", or both?

Reply:
resolved in the new version, there are two pid columns, one for in and one for out